go.mod

```diff
@@ -166,6 +166,9 @@ replace (
 	// TODO Remove it: https://github.com/cosmos/cosmos-sdk/issues/10409
 	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.8.1
 	github.com/gogo/gateway => github.com/notional-labs/gateway v1.1.1-0.20220417180718-8e60e17a098d
+
+	// TODO: remove this when https://github.com/tendermint/tm-db/pull/296 got merged
```

Comment:
Should your PR be put as draft then till this is merged?

Reply:
sure